bin/deploy-dockerhub.sh

```diff
@@ -21,6 +21,20 @@ function retry() {
     return 0
 }
 
+# Usage: push_container_to_repo CONTAINER REPO TAG
+# Push container to specified repo with tag
+# Example: push_container_to_repo remotesettings mozilla/remote-settings latest
+function push_container_to_repo() {
+    container=$1
+    repo=$2
+    tag=$3
+    docker tag $container "$repo:$tag" ||
+        (echo "Couldn't tag $container as $repo:$tag" && false)
```

Comment:
This works, and is equivalent to what we had before.

if we want to improve that, we should exit if this fail instead of just echoing. Something like:

```code
docker tag $container "$repo:$tag"
res=$?
if [ $res -ne 0 ] ; then
    echo "Couldn't tag $container as $repo:$tag"
    return $res
fi
```
But that may be a bit zealous

Reply:
Yeah, maybe we can look through this file in another PR and fix things like this -- I mostly just tried to keep things consistent with what we were already doing in this PR.